src/controllers/user.controller.js

```diff
@@ -8,10 +8,15 @@ const { omit, compact } = require('lodash')
 const bcrypt = require('bcrypt-nodejs')
 
 exports.create = async (req, res, next) => {
-  try {
+ 
+    try{
+      if (req.body.name) req.body.name = req.body.name.split(' ').map((word) => {
+        return word[0].toUpperCase()+word.substr(1).toLowerCase() }).join(' ')
```

Comment:
can you please make this a single line. I see two lines here

if this is more than one line

use

```
if (condition) {
// code
}
```
style